code/API_definitions/qod-api.yaml

```diff
@@ -1201,6 +1210,61 @@ components:
             message: "Service unavailable"
 
   examples:
+    SESSION_AVAILABLE_EXAMPLE:
+      summary: QoS session status is available
+      value:
+        duration: 86400
+        device:
+          ipv4Address:
+            publicAddress: 84.125.93.10
```

Comment:
Is this a safe public IP to use?  It looks like it's in Madrid.

I usually use the Google dns server `8.8. 8.8` since it's one of the best know IPs.

Reply:
There is even an RFC for that 😄 

https://datatracker.ietf.org/doc/rfc5737/
> 3.  Documentation Address Blocks
>  The blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2),
>   and 203.0.113.0/24 (TEST-NET-3) are provided for use in
>  documentation.

Reply:
I have used now 203.0.113.0 for that.
Shell this also be used for applicationServer - IP example or is this okay like it is?

Reply:
@maxl2287 for me it is ok like it is. `198.51.100.0` is also not within a private address range and also reserved for documentation. And it is good that device and applicationServer are in the example in different networks ... which is by the way the reason IANA has reserved three IP ranges for documentation ... to allow examples with multiple networks.